Source/ASConfigurationInternal.mm

```diff
@@ -103,12 +99,12 @@ + (void)test_resetWithConfiguration:(ASConfiguration *)configuration
 
 @end
 
-BOOL ASActivateExperimentalFeature(ASExperimentalFeatures feature)
+BOOL _ASActivateExperimentalFeature(ASExperimentalFeatures feature)
```

Comment:
Should we call this `ASActivateExperimentalFeatures`, add an "s" at the end. I think that makes it more clear that we are passing in multiple features.